rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingServiceImpl.scala

```diff
@@ -173,7 +173,7 @@ trait CachedFindRuleNodeStatusReports extends ReportingService with CachedReposi
 
       for {
         // disabled nodes are ignored
```

Comment:
Comment not accurate!

(but "ignored Nodes are ignored" may be not interesting ...)

Reply:
yeah... not sure what I can use in place of that.

Reply:
So that's ok for you?